src/mscorlib/shared/System/ReadOnlySpan.cs

```diff
@@ -175,7 +175,7 @@ public bool IsEmpty
             }
         }
 #else
-        public T this[int index]
+        public ref readonly T this[int index]
         {
```

Comment:
Nit: You can move the `#if PROJECTN` ifdef back here (so that it looks similar to Span.cs).